.github/workflows/tests.yaml

```diff
@@ -80,6 +80,11 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y python3-pip python3-setuptools python3-wheel python3-venv libapt-pkg-dev
+      - name: Setup LXD
+        uses: canonical/setup-lxd@v0.1.1
+        with:
+          channel: latest/stable
```

Comment:
I believe the guideline from the LXD team is to use either `5.21/stable` or use the default channel (which is also `5.21/stable`)

Reply:
Thanks. I grabbed this from [craft-application](https://github.com/canonical/craft-application/blob/c8e923a45d0277f5042d588743e5d010bb8b0509/.github/workflows/tests.yaml#L63-L66) so do we want to change it there too?

Reply:
yeap, probably!

Reply:
https://github.com/canonical/craft-application/pull/461